src/nbl/video/COpenGLCommandBuffer.h

```diff
@@ -262,8 +262,7 @@ namespace impl
     };
     _NBL_DEFINE_SCMD_SPEC(ECT_WAIT_EVENTS)
     {
-        uint32_t eventCount;
-        core::smart_refctd_ptr<IGPUEvent>* events;
+        core::smart_refctd_ptr<IGPUEvent> event;
```

Comment:
why a single event and not multiple?

Reply:
cmdbuf->waitEvents () pushes multiple of those structs into commands vector

Reply:
you can probably get away with not having any event at all right now

Reply:
yep, those are for future purposes

Reply:
in the future you should implement the commandbuffer as a forward linked list made of command nodes allocated from growable pool allocators ;)

Reply:
yea I remember the idea from one of last talks